source/locale/pt_BR/LC_MESSAGES/internal-communication.po

```diff
@@ -54,21 +52,15 @@ msgstr "DeviceManager: armazenamento de dispositivo e modelo."
 
 #: ../../source/internal-communication.rst:51
 msgid "Persister: component that stores all device-generated data."
-msgstr ""
-"Persister: componente que armazena todos os dados gerados pelo "
-"dispositivo."
+msgstr "Persister: componente que armazena todos os dados gerados pelo dispositivo."
 
 #: ../../source/internal-communication.rst:52
 msgid "History: component that exposes all device-generated data."
 msgstr "History: componente que expõe todos os dados gerados pelo dispositivo."
 
 #: ../../source/internal-communication.rst:53
-msgid ""
-"DataBroker: deals with subjects and Kafka topics, as well as socket.io "
-"connections."
-msgstr ""
-"DataBroker: lida com assuntos e tópicos do Kafka, além de conexões "
-"socket.io."
+msgid "DataBroker: deals with subjects and Kafka topics, as well as socket.io connections."
+msgstr "DataBroker: lida com assuntos e tópicos do Kafka, além de conexões socket.io."
```

Comment:
Since *subject* is a term, we shouldn't translate it. I think it can be changed to:
lida com \*subjects\*